runtime/src/qio/qio_formatted.c

```diff
@@ -2231,257 +2331,170 @@ int _ftoa(char* restrict dst, size_t size, double num, int base, bool needs_i, c
     shownegative = isnegative;
   }
 
+  showbase = 0;
+  if( style->prefix_base && base == 16 && !isnan(num) && !isinf(num) ) {
```

Comment:
So down here we don't need to worry about base being something other than 10 or 16?  Would you please add a comment about that assumption in case we end up wanting floating point octal/binary literals?

(Wasn't there someone interested in adding them, or am I misremembering?)